src/foundry/common/documents/token.d.mts

```diff
@@ -53,7 +53,7 @@ declare abstract class BaseToken extends Document<"Token", BaseToken.Schema, any
 
   /**
    * The default icon used for newly created Token documents
-   * @defaultValue `CONST.DEFAULT_TOKEN` (`"icons/svg/mystery-man.svg"`)
+   * @defaultValue `CONST.DEFAULT_TOKEN`
```

Comment:
I personally don't mind human-readable values for strings but I know that's in conflict with my stance about numberic enums.

I guess I view enums as encapsulated; exposing them as `0`/`1`/whatever is bad practice.